src/WebAppDIRAC/WebApp/handler/AccountingHandler.py

```diff
@@ -229,7 +229,7 @@ def web_getPlotImg(self, file=None, **kwargs):
 
         tempFile.seek(0)
         data = tempFile.read()
-        return FileResponse(data, plotImageFile.encode(), "png")
+        return FileResponse(data, plotImageFile, "png")
```

Comment:
Can you explain this change? I'm suspicious that you're seeing a difference between how DEncode/JEncode behave.

Reply:
This change does not apply to `DEncode`/ `JEncode`.
The file name is obtained as `str`, and `FileResponse` also expects the type `str`, so `.encode()` is not required here.

`FileResponse` will convert this to bytes as needed(e.g.: for **md5**)

Reply:
Additional explanations may be needed here. WebApp handlers inherit `TornadoREST` and when you describe an argument in the target method (eg: `def web_cats (self, color):`) then you will have access to this value. The value type corresponds to the annotation (by default its `str`).